client/valor/coretypes.py

```diff
@@ -158,36 +156,6 @@ def __init__(
         """
         super().__init__(datum=datum, annotations=annotations)
 
-        # validation
-        for annotation in self.annotations:
```

Comment:
removed these checks as they seemed unnecessary given we already check for these things on the API side